docs/tutorials/end_to_end_example.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%timeit
</pre><p>I think we want to do <code>%%time</code> instead of <code>%%timeit</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Woops! Good catch.</p>